### ISSUE TITLE
Resolution RIB does not seem to work right

### ISSUE THREAD

**Issue by Volcanon-** [batfish/batfish#4297]
set routing-options resolution rib inet.0 import LIMIT-NH-RECURSION
set policy-options policy-statement LIMIT-NH-RECURSION term ACCEPT-IPV4-SLASH-32 from route-filter 0.0.0.0/0 prefix-length-range /32-/32
set policy-options policy-statement LIMIT-NH-RECURSION term ACCEPT-IPV4-SLASH-32 then accept
set policy-options policy-statement LIMIT-NH-RECURSION term REJECT-ALL then reject

We have a policy that looks like the above, however I still see /20's showing up in my LPM in Batfish

**Comment by progwriter** [comment]
Yes, customizing rib resolution is a rather niche juniper feature, so this line 
`set routing-options resolution rib inet.0 import LIMIT-NH-RECURSION` 
is not yet supported.
